README.md

```diff
@@ -1 +1,59 @@
-# Catalog-of-my-things
\ No newline at end of file
+# catalog-of-my-things
+An console app t o add resources(`music albums`, `authors`, `genres` and `books`) to`json` files and list them
+
+## Built With
+Ruby
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+- Ruby
+
+### Install
+
+1. Go to the terminal and `cd` into the directory of your preference.
+2. Run `git clone git@github.com:basitali111/Catalog-of-my-things.git` to clone the app.
+3. Open the project in a code editor of your choice that is configured to use Ruby.
+4. Run `ruby main.rb` to start the app
+
+### Run tests
+1. Run `gem install rspec` to install the `RSpec testing framework`
+2. Run `rspec spec` to test all the test files at once
+3. Run `rspec spec/name_of_test_file.rb` to test a particular file
+
+## Brief overview
+
+[Google Drive Link](https://drive.google.com/file/d/1gE7q8keN27T1bqEtkoKV29QPnJ3ZlZ0F/view?usp=sharing)
+
+## Authors
+
+👤 **Basit Ali**
+
+- GitHub: [@githubhandle](https://github.com/basitali111)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/basit-ali-jobs/)
+- Twitter: [Twitter](https://twitter.com/BasitAl35031734)
+
+👤 **Alphayo Wakarindi**
+
+- GitHub: [@githubhandle](https://github.com/alphayowakarindi)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/alphayo-wakarindi-15a825236/)
+- Twitter [Twitter](https://twitter.com/alphayowakarind)
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/basitali111/Catalog-of-my-things/issues).
+
+## Show your support
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+Hat tip to anyone whose code was used
+Inspiration
+etc
+## 📝 License
+This project is MIT licensed.
+
+NOTE: we recommend using the MIT license - you can set it up quickly by using templates available on GitHub. You can also use any other license if you wish.
```

Comment:
- [ ] OPTIONAL You have mentioned about MIT license, but there is no MIT.md file in your repo. Please, fix it before merging to the main branch 👍

Reply:
Hi @elmar8287, thank you so much for taking the time to leave this excellent review. We have made the change and we really appreciate your input to make our code better. Cheers!

Reply:
@alphayowakarindi Happy coding! 🥇